install/space_checker_utils.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import division
+import re
+import os
+import subprocess
+
+def get_wget_download_size(command):
+    """Get the whole download size.
+
+    :param str command: Wget command.
+
+    :return: The actual download size.
+    :rtype: int
+    """
+    try:
+        _, args = command.split(' ', 1)
+    except IndexError:
+        return 0
+
+    download_size_command = "wget --spider --server-response " + args
+    proc = subprocess.Popen(download_size_command.split(), stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT)
+
+    for line in proc.stdout:
+        try:
+            line = line.strip()
+            key, value = line.split(':', 1)
+        except (IndexError, ValueError):
+            continue
+        if key == "Content-Length":
+            try:
+                size = int(value)
+                return size
+            except ValueError:
+                break
+    return 0
+
+def size_to_str(size):
```

Comment:
IMO this function should take a size in bytes, and return to highest unit possible. It should try to cast it as a string.
Then when you call it in ``wget_wrapper`` you use ``%d`` or ``format()`` to convert it to a string. (You would need to rename the function to something like ``byte_to_human()``).

This would be nicer that way IMO.